BUILD.gn

```diff
@@ -102,6 +102,8 @@ core_validation_sources = [
   "layers/image_state.cpp",
   "layers/pipeline_state.h",
   "layers/pipeline_state.cpp",
+  "layers/pipeline_layout_state.cpp",
```

Comment:
I think if this part gets pulled into [#3958 ](https://github.com/KhronosGroup/Vulkan-ValidationLayers/pull/3958) it would pass CI and could be merged right away.